### ISSUE TITLE
Can't start multiple TestxServer at the same time

### ISSUE THREAD

**Issue by b1zzu** [aerogear/graphql-testx#47]
I would like to use graphql-testx with parallel tests, but if I try to do it they will collead on the same port.

This code simulates my issue:
```js
const server1 = new TestxServer(ITEM_MODEL);
const server2 = new TestxServer(ITEM_MODEL);

// start both servers at the same time
await Promise.all([server1.start(), server2.start()]);
```

**Comment by vinifarias** [comment]
@b1zzu for some reason this problem doesn't occur in code implemented in #45 :squirrel:

**Comment by b1zzu** [comment]
@vinifarias I've added a test for this bug https://github.com/aerogear/graphql-testx/blob/867d33c1d3d4586684f3cc03b19d6796cde745f6/src/index.test.ts#L31 so if it doesn't occur in #45 we can remove the skip from the test and it should pass, otherwise keep in mind that is a low priority bug, I've created it just to track it and don't forget it
